### ISSUE TITLE
Typo: the last character of not-found plugin vanished

### ISSUE THREAD

**Issue by znagadeon** [eslint/eslint#13885]
**Tell us about your environment**

* **ESLint Version:** v7.14.0
* **Node Version:** v12.13.0
* **npm Version:** v6.14.1

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
  "extends": [
    "plugin:not-installed-plugin"
  ]
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
Oops! Something went wrong! :(

ESLint: 7.14.0

ESLint couldn't find the plugin "eslint-plugin-not-installed-plugi".

(The package "eslint-plugin-not-installed-plugi" was not found when loaded as a Node module from the directory "/Users/kakao/src/homepage".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-not-installed-plugi@latest --save-dev

The plugin "eslint-plugin-not-installed-plugi" was referenced from the config file in ".eslintrc.json".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

**What did you expect to happen?**
Plugin name would be displayed properly.

**What actually happened? Please include the actual, raw output from ESLint.**
The last character is vanished.

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by mdjermanovic** [comment]
Hi @znagadeon, thanks for the issue!

`"extends": ["plugin:foo"]` is actually an invalid configuration.

It should be either `"extends": ["plugin:foo/configName"]` if `foo` is a plugin that exports a config, or `"extends": ["foo"]` without `plugin:` if `foo` is a shareable config (not a plugin):

* [Using the configuration from a plugin](https://eslint.org/docs/user-guide/configuring#using-the-configuration-from-a-plugin)
* [Using a shareable configuration package](https://eslint.org/docs/user-guide/configuring#using-a-shareable-configuration-package)

Aside from missing the last character, the error message you're getting is completely wrong. It should be like [this message](https://github.com/eslint/eslint/pull/13473#discussion_r479688611).

We fixed this issue in https://github.com/eslint/eslint/pull/13473, but missed to transfer that change when some functionality was extracted to the `@eslint/eslintrc` package.

I'm closing this as a duplicate of https://github.com/eslint/eslintrc/issues/12
